operator/controllers/service_controller.go

```diff
@@ -68,50 +69,29 @@ func (r *ServiceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return reconcile.Result{}, fmt.Errorf("could not fetch Service: %+v", err)
 	}
 
-	if service.IsCompleted() {
-		return reconcile.Result{}, nil
-	}
-
 	logger.Info("Reconciling Service", "service", service.Name)
 
-	if service.IsServiceRunNotCreated() {
-		logger.Info("Service Run does not exist, creating")
-
-		serviceRunName, err := r.createServiceRun(ctx, service)
+	serviceRun := &v1alpha1.ServiceRun{}
```

Comment:
please add some comments that explain the current behaviour so we don't get confused again after the weekend ;)